routes/user.js

```diff
@@ -89,63 +90,91 @@ router.post("/setUserInfo/:userId", async (req, res) => {
   }
 });
 
+router.post(
+  "/profileImage/:userId",
+  multerUploads,
+  requireUserAuth,
+  async (req, res) => {
+    try {
+      const imageUploadRes = await uploadImagesToAWS(
+        req.files["image"],
+        "profileImg"
+      );
+      const imgUrl = imageUploadRes[0];
+
+      const userToUpdate = await User.findOneAndUpdate(
+        { _id: req.params.userId },
+        { $set: { profileImg: imgUrl } },
```

Comment:
Haven't been able to get this to set on my end. The route is being called but the database document remains unchanged when I try to upload a pfp

Reply:
Seems like the user model didn't get pushed.  Merged + fixed and pushed.

Reply:
Awesome will test and get back tonight 👍